sr_example/scripts/sr_example/commander/sr_right_hand_partial_traj_grasp.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import rospy
+from sr_robot_commander.sr_hand_commander import SrHandCommander
+from trajectory_msgs.msg import JointTrajectory, JointTrajectoryPoint
+
+rospy.init_node("set_traj_goals", anonymous=True) 
+rospy.sleep(1)  # Do not start with zero
+
+def construct_trajectory_point(posture, duration):
+    trajectory_point = JointTrajectoryPoint()
+    trajectory_point.time_from_start = rospy.Duration.from_sec(float(duration))
+    trajectory_point.positions = []
```

Comment:
This shouldn't be necessary, as the constructor of JointTrajectoryPoint does it already.